src/main/java/org/jabref/logic/ai/ingestion/storages/MVStoreFullyIngestedDocumentsTracker.java

```diff
@@ -1,59 +1,54 @@
-package org.jabref.logic.ai.embeddings;
+package org.jabref.logic.ai.ingestion.storages;
 
-import java.util.HashSet;
+import java.nio.file.Path;
 import java.util.Map;
 import java.util.Optional;
-import java.util.Set;
 
-import com.google.common.eventbus.EventBus;
-import org.h2.mvstore.MVStore;
+import org.jabref.gui.DialogService;
+import org.jabref.logic.ai.ingestion.FullyIngestedDocumentsTracker;
+import org.jabref.logic.ai.util.MVStoreBase;
+import org.jabref.logic.l10n.Localization;
 
 /**
  * This class is responsible for recording the information about which documents (or documents) have been fully ingested.
  * <p>
- * It will also post an {@link DocumentIngestedEvent} to its event bus when a document is fully ingested.
```

Comment:
Carl will be happy to see this removed ^^

Reply:
Ah, I thought this comment is the PR and I should remove it.

But actually I already removed :) (red background means removed in PR)